pkg/util/util.go

```diff
@@ -171,8 +171,8 @@ func CurrentServer() (string, error) {
 	return server, nil
 }
 
-func DecorateErrorWithDocsLink(err error) error {
-	return fmt.Errorf("%s\nfor more information: %s", err.Error(), store.Get().DocsLink)
+func DecorateErrorWithDocsLink(err error, link string) error {
```

Comment:
instead of breaking all calls to this function you should have made this optional by using `(err error, link ...string)`. Then you can pass the link arg only when needed

Reply:
but without link, this function is not decorating anything!

Reply:
it is, `store.Get().DocsLink` would be the default.